src/coreclr/vm/hillclimbing.cpp

```diff
@@ -1,443 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-//=========================================================================
-
-//
-// HillClimbing.cpp
-//
-// Defines classes for the ThreadPool's HillClimbing concurrency-optimization
-// algorithm.
-//
-
-//=========================================================================
-
-//
-// TODO: write an essay about how/why this works.  Maybe put it in BotR?
-//
-
-#include "common.h"
-#include "hillclimbing.h"
```

Comment:
Delete `hillclimbing.h` too.

Also, delete all deleted files from https://github.com/dotnet/runtime/blob/main/src/coreclr/vm/CMakeLists.txt. It is going to fix some of the build breaks.